src/server/sv_console.c

```diff
@@ -20,10 +20,23 @@
  */
 
 #include <signal.h>
-#include <curses.h>
 
 #include "sv_local.h"
 
+// Visual Studio hack
+#if defined(_MSC_VER)
+#undef MOUSE_MOVED
+#undef bool
+#endif
+
+#include <curses.h>
+
+// Visual Studio hack
+#if defined(_MSC_VER)
+#undef bool
+#define bool _Bool
+#endif
+
```

Comment:
pdcurses breaks the definition for bool. It's an ugly fix, and I didn't want to edit the third party lib so we can update if need be.